timml/besselaesnew/besselaesnew.py

```diff
@@ -0,0 +1,706 @@
+import numba
+import numpy as np
+
+
+def initialize():
+    pass
+
+
+@numba.njit
+def make_rbnimon():
+    rrange = np.arange(9.0)
+    rbinom = np.zeros((9, 9))
+    for n in range(9):
+        for m in range(9):
+            if m > n:
+                rbinom[n, m] = 1 
+            else:
+                rbinom[n, m] = np.prod(rrange[m + 1: n + 1]) / np.prod(
+                    rrange[1 : n - m + 1]
+                )
+    return rbinom
+
+
+RBINOM = make_rbnimon()
+AC = np.zeros(9)
```

Comment:
I think it could be a bit more readable to just define these arrays at once like:
```Python
    AC = np.array([-0.5...e0, -0.12...e3, ...])
```
Do you see significant value to assign the value interleaved to this predefined array?